src/components/Node.tsx

```diff
@@ -261,7 +270,7 @@ export default class NodeComp extends React.Component<NodeProps, NodeState> {
                 }
             }
 
-            // add our exit localizations
+            /** Account for localized exits */
```

Comment:
Curious the purpose for this commenting style

Reply:
We use jsDoc comments elsewhere so just trying to make things consistent. Not something I'm passionate about either way.